x-pack/plugins/security/server/lib/authorization/disable_ui_capabilities.test.ts

```diff
@@ -88,6 +88,7 @@ describe('usingPrivileges', () => {
               indices: true,
             },
           },
+          catalogue: {},
```

Comment:
All of these test are using an empty `catalogue` entry, and `disable_ui_capabilities` is treating it like any other ui capability. We have logic in `disable_ui_capabilities` to only disable "navLinks" that are registered for a feature, on the surface it seems like we'd want similar logic for "catalogue" entries.

Reply:
`navLinks` is special because we know about every nav link that Kibana has, even if that nav link isn't registered for a feature.

Catalogue entries, much like management entries, are registered client-side, so the Kibana server doesn't have a complete list of them. The only catalogue entries we deal with are the ones explicitly registered via features.